html/pfappserver/root/config/portal_module/view.tt

```diff
@@ -15,6 +15,17 @@
           <option value="[% access_duration %]"> [% access_duration %] </option>
        [% END %]
      </select>
+     <select name="on_failure_action_match" id="on_failure_action_match" data-required="required">
```

Comment:
It would be better to have the on_success and on_failure root modules be tied as a parameter of the action directly in the root module instead of being tied to the connection profile

Reply:
Not sure to understand what you mean.